Doppler.Sap/Factory/CreditNoteHandler.cs

```diff
@@ -112,7 +112,7 @@ public async Task<SapTaskResult> UpdatePaymentStatusHandle(SapTask dequeuedTask)
 
                 if (dequeuedTask.CreditNoteRequest.Type != (int)CreditNoteEnum.Refund || !dequeuedTask.CreditNoteRequest.TransactionApproved)
                 {
-                    _logger.LogError("The outgoing payment couldn't create in SAP because the type of the credit note  must be 'Refund' and the transaction approved..");
+                    _logger.LogError($"The outgoing payment couldn't create in SAP because the type of the credit note  must be 'Refund' and the transaction approved: '{sapResponse.SapResponseContent}'");
```

Comment:
In general, we try to use [structured logging](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-5.0#log-message-template)

```suggestion
                    _logger.LogError("The outgoing payment couldn't create in SAP because the type of the credit note  must be 'Refund' and the transaction approved: '{SapResponseContent}'", sapResponse.SapResponseContent);
```